### DISCUSSION TITLE
Bottom label pushed up and restored

### DISCUSSION THREAD

**Discussion by TallNutAlt** [react-navigation/react-navigation#12322] [Q&A]
![image](https://github.com/user-attachments/assets/1367f7a8-b8b6-4edc-a75a-39c406bfda0b)
At first, it was like this
![image](https://github.com/user-attachments/assets/9d74a396-9920-4949-86d3-767615b082fd)
Afterwards, it will recover in a flash. May I ask what the situation is? I need help

**Comment by satya164** [comment]
I have seen similar issue with react-native-screens. Are you using the tabs nested inside native stack?

Alternatively it could also be an issue with safe area detection. I recommend using https://github.com/zoontek/react-native-edge-to-edge

**Reply by TallNutAlt** [reply]
> I have seen similar issue with react-native-screens. Are you using the tabs nested inside native stack?
> 
> Alternatively it could also be an issue with safe area detection. I recommend using https://github.com/zoontek/react-native-edge-to-edge

Yes, I nested it within react navigation/native stack

**Reply by TallNutAlt** [reply]
> I have seen similar issue with react-native-screens. Are you using the tabs nested inside native stack?
> 
> Alternatively it could also be an issue with safe area detection. I recommend using https://github.com/zoontek/react-native-edge-to-edge

Thank you for using React native edge to edge. It works normally
